meson.build

```diff
@@ -1139,6 +1139,17 @@ else
 endif
 conf.set10('HAVE_P11KIT', have)
 
+want_fido2 = get_option('fido2')
+if want_fido2 != 'false' and not skip_deps
+        libfido2 = dependency('libfido2',
+                                required : want_fido2 == 'true')
+        have = libfido2.found()
+else
+        have = false
+        libfido2 = []
+endif
+conf.set10('HAVE_FIDO2', have)
```

Comment:
You're defining for `HAVE_FIDO2` here, and checking `HAVE_LIBFIDO2` below. Please change that to ` `HAVE_LIBFIDO2` everywhere to match the dependency name.